server/src/api/listings.ts

```diff
@@ -55,4 +56,26 @@ listingRouter.get(
   }
 );
 
+listingRouter.post(
+  '/',
```

Comment:
So this path is used for both getting and adding listings?

Reply:
@chshda Yep but they have different HTTP request methods.